client/index.html

```diff
@@ -329,6 +501,15 @@ <h2>
           </div>
 
           <!-- Loadout & settings buttons -->
+
+           <div style="display: flex; flex-direction: row; gap: 15px">
+            <!-- login button -->
+             <button class="btn btn-lg btn-darken btn-primary" id="btn-login">
+              <i class="fa-solid fa-right-to-bracket "></i>
+              <span translation="Login"></span>
```

Comment:
all of these translation attributes are incorrectly used, and not supplemented in `client/src/translations` with respective values.